tag_hierarchy/commands/search.cpp

```diff
@@ -0,0 +1,103 @@
+//
+// Created by Petter Moe Kvalvaag on 2019-10-04.
```

Comment:
I guess this date is left from another file?

Reply:
I think that is actually when this was started, better late than never 😆 
On the topic, IDEs tend to leave these things but I'd rather delete them, agree?

Reply:
Aha! I am fine with either way :)